### ISSUE TITLE
Are there C bindings for codespan?

### ISSUE THREAD

**Issue by aminya** [brendanzab/codespan#336]

**Comment by Johann150** [comment]
No, there are currently no C bindings. I don't think there are plans to add some.

**Comment by aminya** [comment]
C bindings for such a library is important because other languages are not written in Rust. If the main functions can be identified, then I think cbindgen can be used.
